src/common/components/Popover/Popover.theme.ts

```diff
@@ -0,0 +1,38 @@
+import { type VariantProps, tv } from "tailwind-variants";
+
+export type PopoverVariants = VariantProps<typeof popoverTheme>;
+
+export const popoverTheme = tv(
+  {
+    slots: {
+      popoverContent: [
+        "bg-bg-base",
+        "z-50",
+        "w-72",
+        "rounded-md",
+        "border",
+        "p-4",
+        "shadow-md",
+        "outline-none",
+        "text-popover-foreground",
+        "data-[state=open]:animate-in",
+        "data-[state=closed]:animate-out",
+        "data-[state=closed]:fade-out-0",
+        "data-[state=open]:fade-in-0",
+        "data-[state=closed]:zoom-out-95",
+        "data-[state=open]:zoom-in-95",
+        "data-[side=bottom]:slide-in-from-top-2",

```

Comment:
We added these but I don't see the equivalent animations in tailwind config or css files. Could you add them in too?

Reply:
@Aztriltus

I left out adding new tailwind configurations since it would probably be best to standardize them in a separate PR

seems like shadcn uses this `[tailwind-animate](https://github.com/jamiebuilds/tailwindcss-animate)` plugin for animation, will we be using the same plugin for animation? if that's the case I can add it in for this PR!

Reply:
I think let's add the plugin! At least things are working as coded, and not missing features!

Reply:
added at 1e136c7!